Monika After Story/game/script-topics.rpy

```diff
@@ -15413,3 +15413,75 @@ label monika_why_do_you_read:
+    m "The genre is kind of unfamiliar territory to me though, but I thought I'd hear if you were interested in it."
+    m "It's called tabletop roleplaying! Have you heard of it?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "It's called tabletop roleplaying! Have you heard of it?"
+        "Yes.":
+            m "Right, just please indulge me for a second then."
+
+        "No.":
+            m "Oh really? Then please indulge me while I tell you about it."
+
+    m "You may have played 'pretend' when you were a kid growing up, right?"
+    m "It's a game where you assume the role of a different person, maybe in another world, and act and speak as them to form a narrative."
+    m "When you're doing it with friends, it becomes much more fun, as you are also met with other people's character, and you interact with each other."
+    m "But 'pretend' always had one big problem{w=0.3}...What do you do when you can't agree on the story's plot?"
+    m "It could often devolve into a 'my character is invulnerable and always wins' kind of situation, you know?"
+    m "To counteract this problem, tabletop roleplaying was invented."
+    #Not sure if I should include this part about Dungeons and Dragons, since it's a pop culture reference, kind of. Though I think a topic about TRPGs isn't really complete without it either.
+    m "The most popular tabletop roleplaying game is called 'Dungeons & Dragons', or D&D, but that's far from the only one that exists, and its narrative form kind of always assumes a medieval fantasy setting."
+    m "In D&D, not all players are equal, since there necessarily always needs to be a 'Dungeon Master', basically someone who makes the setting of the story and controls the monsters and other obstacles."
+
+    m "Other Tabletop RPGs are a lot like theatre in the sense that everyone plays a character in the story, aside from the fact that the story itself is being improvised as it goes."
+    m "Everyone participating is taking turns, writing a story by reacting to the others' actions and acting on it."
+    m "I guess that makes it a good writing exercise..."
+    m "You don't need to think of the entire story by yourself, but at the same time, your character will end up in situations you probably never could anticipate..."
+    m "...So you have to improvise at all times by choosing what you think would be the best course of action in every situation, while still staying in-character."
+
+    m "Unlike most other forms of literature, it's not an author expressing their own thoughts and opinions through written text."
+    m "The best way I can explain it, is that tabletop roleplaying is kind of like the multiplayer experience of literature."
+    m "Just a little video game reference for you~"
+    m "The only problem with that though, is that once you reach the end of the story, if you even manage to get to a point in the story that you can call an ending..."
```

Comment:
I'd shorten this so you can bring in the next line, flows better imo:
`The only problem is that once you reach the end of the story--{w=0.2}if you even manage to reach that point,{w=0.3} is that you'll often end up with a big, almost unreadable mess."`

Reply:
I hope it's okay that I split these into two sentences, instead of a comma after "that point", I'll just skip to the next sentence and make "You'll often" the beginning of the next one, skipping "is that".

Reply:
`The only problem is that once you reach the end of the story, {w=0.2}if you even manage to reach that point..."
"You'll often end up with a big, almost unreadable mess."`

Reply:
I think you should add '...' at the begining of your second line then (not sure). But yeah, why not.

Reply:
I don't know if ellipsis at the end and beginning of the next sentence is proper form though.

Reply:
Yup, you're right, nvm then.